git/git.go

```diff
@@ -119,3 +116,20 @@ func (repo *Repository) GitCommand(callerArgs ...string) *exec.Cmd {
 func (repo *Repository) GitDir() string {
 	return repo.gitDir
 }
+
+// GitPath returns that path of a file within the git repository, by
+// calling `git rev-parse --git-path $relPath`. The returned path is
+// relative to the current directory.
+func (repo *Repository) GitPath(relPath string) (string, error) {
```

Comment:
Good idea! There is a typo in the commit message introducing this method where the second time it is mentioned, it is misspelled as `GetPath()`. Maybe that could be fixed before merging?

Reply:
Done :heavy_check_mark: